src/reference-manual/statements.qmd

```diff
@@ -361,6 +361,38 @@ including arrays of vectors and matrices.   For container arguments,
 their sum will be added to the total log density.
 
 
+## Increment log density with a change of variables adjustment
+
+
+A variant of the `target +=` statement described above is the
+`jacobian +=` statement. This can be used in the transformed parameters block
+or in functions ending with `_jacobian` to mimic the log Jacobian
+adjustments accrued by built-in variable transforms.
+
+Similarly to those implemented for the built-in transforms, these Jacobian adjustment
+may be skipped for optimization.
```

Comment:
I would say "may be skipped for maximum likelihood estimation via optimization" to make it clear the optimization isn't speed related here.  If you leave the transforms and optimize, you get the MAP (max a posteriori) estimate.